plugins/chat/assets/javascripts/discourse/components/channels-list-direct.gjs

```diff
@@ -46,6 +48,32 @@ export default class ChannelsListDirect extends Component {
     return this.chatChannelsManager.directMessageChannels?.length === 0;
   }
 
+  get publicMessageChannelsEmpty() {
+    return (
+      this.chatChannelsManager.publicMessageChannels?.length === 0 &&
+      this.chatStateManager.hasPreloadedChannels
+    );
+  }
+
+  get publicChannelsDisabled() {
+    if (!this.siteSettings.enable_public_channels) {
```

Comment:
I don't think we need the `!` here, if enable public channels is true then `publicChannelsDisabled` should be false.